_includes/header.html

```diff
@@ -12,6 +12,9 @@ <h2>
         <a href="//docs.parseplatform.org/parse-server/guide/" target="_blank">
             <button class="large">Get Started</button>
         </a>
+        <a href="https://community.parseplatform.org" target="_blank">
```

Comment:
remove the protocol, pretty sure that'll work right cross domain.

Reply:
Right you are 👍